cmd/ocitools/generate.go

```diff
@@ -382,3 +406,15 @@ func setupLinuxNamespaces(g generate.Generator, needsNewUser bool, nsMaps map[st
 		g.AddOrReplaceLinuxNamespace(nsName, nsPath)
 	}
 }
+
+func onlyExportFlagSpecified(context *cli.Context) bool {
+	onlyExportFlags := []bool{
+		context.Bool("seccomp-only"),
+	}
+	for _, flag := range onlyExportFlags {
+		if flag {
+			return true
```

Comment:
On Thu, Jul 21, 2016 at 02:42:22PM -0700, Antonio Murdaca wrote:
> No, strongly disagreeing here to pass down the context - what you're
> referring to is only being done for Request scoped stuff and I'm not
> sure we have any request here…

I'm pretty sure there aren't HTTP requests in ocitools, but the
command-line call creates a similar Context, and I don't see a reason
to distinguish between “serving an HTTP request”, “serving a CLI
request”, and “serving a library API request”.  What I'm looking for
is something like Python's **kwargs [1] for tunneling
low-level-function config through an intermediate.  If there's a more
canonical Go way of doing that, it would be fine too.  But exposing
seccomp_only (and similar stuff if we add more in the future) as
separate parameters in SaveToFile just seems like busywork.

[1]: https://docs.python.org/3/faq/programming.html?highlight=kwargs#how-can-i-pass-optional-or-keyword-parameters-from-one-function-to-another